### PR TITLE
EKO v0.13.1 compatibility

### PR DESCRIPTION
**Pull request by felixhekhorn**
- [x] fix EKO version to 0.13.1

### CODE REVIEW

**Comment by AleCandido** [comment]
Currently, tests are failing because of the regression operator card used has not been updated for v0.13.x.

https://raw.githubusercontent.com/NNPDF/theories/main/data/operator_cards/400/_template.yaml

File: src/pineko/evolve.py [hunk 1]
```diff
@@ -116,16 +116,16 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
...
-    atlas = ThresholdsAtlas(
-        masses=np.array([tcard["mc"], tcard["mb"], tcard["mt"]]) ** 2,
-        q2_ref=tcard["Q0"] ** 2,
-        nf_ref=tcard["nfref"],
-        thresholds_ratios=np.array([tcard["kcThr"], tcard["kbThr"], tcard["ktThr"]])
-        ** 2,
-        max_nf=tcard["MaxNfPdf"],
+    masses = np.array([tcard["mc"], tcard["mb"], tcard["mt"]]) ** 2
+    thresholds_ratios = np.array([tcard["kcThr"], tcard["kbThr"], tcard["ktThr"]]) ** 2
+    for q in range(tcard["MaxNfPdf"], 6 + 1):
```
Comment (giacomomagni) [inline src/pineko/evolve.py:121]:
```suggestion
    for q in range(tcard["MaxNfPdf"] + 1, 6 + 1):
```
I think here we need to loop only thresholds  higher than the last heavy quark included no?

**Comment by andreab1997** [comment]
> Currently, tests are failing because of the regression operator card used has not been updated for v0.13.x.
> 
> https://raw.githubusercontent.com/NNPDF/theories/main/data/operator_cards/400/_template.yaml
> 
> @andreab1997

Yes today I am starting to change all the benchmark files until all the tests will work again (for both #94 and #88 ).

File: src/pineko/evolve.py (hunk 1 above)
Comment (andreab1997) [reply src/pineko/evolve.py:121]: Yes I agree

**Comment by andreab1997** [comment]
This should be ready now. Tests and benchmarks are passing again. I believe we should merge this as soon as possible and release pineko immediately in such a way we will have a proper `pineline` release for production.

File: src/pineko/evolve.py [hunk 2]
```diff
@@ -173,28 +187,26 @@ def evolve_grid(
...
     sc = eko.couplings.Couplings(
         tcard.couplings,
         tcard.order,
         evmod,
-        quark_masses,
-        hqm_scheme=tcard.quark_masses_scheme,
-        thresholds_ratios=np.power(list(iter(tcard.matching)), 2.0),
+        masses=[(x.value) ** 2 for x in tcard.heavy.masses],
+        hqm_scheme=tcard.heavy.masses_scheme,
+        thresholds_ratios=thresholds_ratios,
```
Comment (giacomomagni) [inline src/pineko/evolve.py:208]:
```suggestion
        thresholds_ratios=thresholds_ratios.tolist(),
```
I you want for future ourself, just to be sure.

**Review by giacomomagni — APPROVED** [review]

**Review by AleCandido — APPROVED** [review]

**Comment by andreab1997** [comment]
Ok I would say that this is done. I am waiting a bit more before merging just to give @felixhekhorn time to have a look if he wants to.

**Review by felixhekhorn — COMMENTED** [review]
I'm fine with it (I can not approve my own PR)

**Pull request merged**